tools/pot/openvino/tools/pot/engines/ie_engine.py

```diff
@@ -303,12 +303,12 @@ def completion_callback(request, user_data):
                 start_time = time()
 
         progress_log_fn = logger.info if print_progress else logger.debug
-        self._ie.set_config({'CPU_THROUGHPUT_STREAMS': 'CPU_THROUGHPUT_AUTO', 'CPU_BIND_THREAD': 'YES'}, self._device)
```

Comment:
I think we should use the try-except condition for this place. We will update this in the future separately. cc @AlexKoff88

Reply:
done